.github/workflows/release.yml

```diff
@@ -147,11 +147,6 @@ jobs:
             type=schedule,pattern=${{ steps.base-release.outputs.base-release }}-{{date 'YYYYMMDD'}}
             type=raw,value=manual,suffix=${{ github.event.inputs.suffix }},enable=${{ github.event_name == 'workflow_dispatch' }}
 
-      - name: Setup npm registries
```

Comment:
I don't think this was actually doing anything

Reply:
Only thing I could think of is a niche npm package one of our repos might be depending on. Afaik everything should be accessible publicly on npmjs.org without this setup. We should be able to add this back if we run into any issues down the road anyway